### ISSUE TITLE
Gameweek start day

### ISSUE THREAD

**Issue by rohanday3** [alan-turing-institute/AIrsenal#651]
Is there currently a way to check the start day of a given game week? I'm working on automating the transfer process to run the pipeline periodically but the game weeks don't start on the same day or time each week.

**Comment by nbarlowATI** [comment]
We don't currently have a command-line way of doing this (though it could be an easy/fun thing for someone to put in), but it's possible via a couple of commands at the python prompt:
```
>>> from airsenal.framework.utils import *
>>> fixtures = get_fixtures_for_gameweek(get_next_gameweek())
>>> sorted([f.date for f in fixtures])
```
should hopefully give an ordered list of fixture times for the upcoming gameweek.
```

**Comment by rohanday3** [comment]
Awesome, I didn't realize that the transfer deadline was always linked to the first match of the gameweek. This makes it alot easier, I was thinking of using crontab to schedule the task and then the task will update the cronjob for the following week. Do you have any advice or suggestions for me?

**Comment by nbarlowATI** [comment]
Yep, that could make sense - would protect against being caught out by midweek or Friday deadlines!   I'm not really an expert on crontab, but it seems a sensible tool for the job, as long as you have a machine that is online at the necessary times.   The other alternative if that isn't the case could be some sort of cloud-based serverless compute (Amazon Lambda or Azure Functions)?
